pages/blog/governance-motivation.md

```diff
@@ -0,0 +1,126 @@
+- we are a large community, but not large enough to handle two different charters, not now.
+
+The other good reason for not splitting is that we believe the initiative should work on spec and tools to provide essential open-source tools that consistently support the latest version of the specification. In other words, when we release the newest version of the specification, you can already use it with official tools and do not have to wait for other tooling providers to catch up. We know big companies do not care much about basic tooling as they most probably write their own as it is much easier for them to work without unnecessary noise. Nevertheless, we need to think about the small ones too. Two TSCs sound like having two silos that work separately, and it is not good for the start. Nobody says, though, that TSC can't organically grow into two separate groups in the future.
+
+
+### Value work more than money
+
+We want to have a TSC consisting of all the active committers in the project, not people who are there because their company is a sponsor of the project. You are TSC member because you work on the project, you code, write docs, maintain CI. You work, and therefore, you are a part of the initiative.
+
+Why? Don't you like the money?
+
+We like money, don't get us wrong here :smiley: Money helps the project a lot, we can sponsor many good things with the money we obtain from you.
+
+Nevertheless, we believe that it isn't good to run a project alone with lots of bucks in the pocket. 
+
+What? We need a doctor here!
+
+:man_shrugging: We just prefer to work with a large group of people, from different cultures, from other parts of the world, with diverse experience and use cases in heads. That is just how you build great products by having many people around you to help and validate the ideas actively.
+
+
+### Respecting committers and the right to vote 
+
+The rule we want to follow is that the committer automatically gets a right to vote. 
+
+Why? 
+
+Voting is always about essential things, essential for the entire initiative. Who else should have a right to vote if not people that are directly affected by decisions of the ToC. The tricky situation here is that this approach can lead to a problem where one company hires most committers and, therefore, takes over project steering. This is something we actually have at the moment. The majority of folks working actively on the project are from Postman. Nothing terrible happened to the project for now, but this doesn't mean the whole community feels comfortable.
+
+We had a tough brainstorming here about finding the right balance between respecting all contributors equally and still making sure that there is no risk that a company has most voters. That is why we have a rule that only 1/4 (1/3 in case we need to look for more maintainers to bring balance back to the force) of voters can be affiliated with the same company. 
```

Comment:
```suggestion
We had a tough brainstorming here about finding the right balance between respecting all contributors equally and still making sure that there is no risk that a company has most voters. That is why we have a rule that only 1/4 of voters can be affiliated with the same company. Or 1/3 in case we need to look for more maintainers to bring balance back to the force.
```